plugins/services/src/js/components/forms/PlacementSection.js

```diff
@@ -0,0 +1,270 @@
+import React, { Component } from "react";
+import { Tooltip } from "reactjs-components";
+
+import { findNestedPropertyInObject } from "#SRC/js/utils/Util";
+import AddButton from "#SRC/js/components/form/AddButton";
+import DeleteRowButton from "#SRC/js/components/form/DeleteRowButton";
+import FieldAutofocus from "#SRC/js/components/form/FieldAutofocus";
+import FieldError from "#SRC/js/components/form/FieldError";
+import FieldHelp from "#SRC/js/components/form/FieldHelp";
+import FieldInput from "#SRC/js/components/form/FieldInput";
+import FieldLabel from "#SRC/js/components/form/FieldLabel";
+import FieldSelect from "#SRC/js/components/form/FieldSelect";
+import FormGroup from "#SRC/js/components/form/FormGroup";
+import FormGroupHeading from "#SRC/js/components/form/FormGroupHeading";
+import FormGroupHeadingContent
+  from "#SRC/js/components/form/FormGroupHeadingContent";
+import FormRow from "#SRC/js/components/form/FormRow";
+import Icon from "#SRC/js/components/Icon";
+
+import OperatorTypes from "../../constants/OperatorTypes";
```

Comment:
This file is in `plugins`, the idea of using aliases was to draw the line between plugins/packages and shared code in `src` so that we can extract common parts into a some sort of SDK/foundation. So a plugin should still get its files via relative paths. Does that make sense?

Reply:
Yeah makes sense! Didn't realize this was how we approached this, thanks for clarifying.